interface/src/app/mlmodel/selector.tpl.html

```diff
@@ -1,8 +1,8 @@
 <div class="alert alert-danger" role="alert" ng-if="$ctrl.errorMessage">
   {{$ctrl.errorMessage}}
 </div>
-<select
-  ng-options="option as option.title for option in $ctrl.availableMlModels"
-  ng-model="$ctrl.selectedMlModel">
-  <option value="">--- Please select a machine learning model ---</option>
```

Comment:
It forces the select box to be *really* wide. If it is necessary on a particular page, I would prefer to see this sort of instruction appear in a form label or tool tip to be implemented by the parent view rather than in the reusable component itself.